about/templates/about/about.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% block title %}Kontakt{% endblock title %}
+
+{% block content %}
+
+    <div class="row">
+
+        <div class="col-md-2 col-sm-1"> </div>
+
+        <div class="col-md-8 col-sm-10">
+
+            <div class = "page-header">
+
+                <h2>O nás a o liahni</h2>
+
+            </div>
+
+            <p>Programátorskú liaheň spáchali organizátori
+                <a href = "http://ksp.sk">Korešpondenčeného seminára z programovania</a>,
+                ktorý je spolu s <a href = "http://kms.sk">KMS</a> a <a href = "http://fks.sk">FKS</a>
+                združený v združení <a href = "http://trojsten.sk">Trojsten</a>.
+                Sme z <a href="http://www.fmph.uniba.sk">Fakulty Matematiky, Fyziky a Informatiky UK</a>
+                v Bratislave, väčšina z nás ešte študuje.
+            </p>
+
+            <p>Tejto verzii predchádzala <a href = "https://liahen.ksp.sk">"stará" Liaheň</a>,
```

Comment:
Tato liahen casom bude ficat na linke liahen.ksp.sk.., tento link by sa mohol vcas upravit na daco v style oldliahen.ksp.sk, alebo odstranit uplne, nech sa na to nezabudne